internal/db/request.go

```diff
@@ -50,7 +50,7 @@ func (db *db) execRequest(ctx context.Context, request string) *client.RequestRe
 	identity := GetContextIdentity(ctx)
 	planner := planner.New(ctx, identity, db.acp, db, txn)
 
-	results, err := planner.RunRequest(ctx, parsedRequest)
+	results, err := planner.RunRequest(ctx, parsedRequest, request.OperationName)
```

Comment:
todo: This appears to be a less than optimal code flow, as the operation is known by this point, and we can discard the operation name and all irrelevant operations.

Unless I have missed something, and the 'extra' ops are not necessarily irrelevant, I don't think the planner should be burdened with this unwanted information (and the hassle of selecting the correct operation).